cli/src/cluster_query.rs

```diff
@@ -768,6 +770,18 @@ pub fn process_catchup(
             )
         };
 
+        let slot_distance_avg = node_slot as i64 - first_node_slot as i64;
```

Comment:
Following, this naming convention (total_sleep_interval), https://github.com/solana-labs/solana/pull/15608/files#r586498430, let's rename this to `total_slot_distance`.